scania-digital-design-system/sdds
# fix(checkbox/radio): adjusted click-area to cover hover effect

**Describe pull-request**  
adjusted click-area to cover hover effect
also adjusted disabled colors
**Solving issue**  
Fixes: [AB#2206](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2206) [AB#2207](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2207) [AB#2202](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2202) [AB#2203](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2203)

**How to test**  
Open storybook and check out the checkbox and radio button component. test if the click area includes the hover effect.
